organisaatio-ui/src/types/types.ts

```diff
@@ -152,3 +165,9 @@ export type SelectOptionType = {
     value: string;
     label: string;
 };
+
+export type LanguagedInputBind = {
+    name: string;
+    value: string;
+    onChange: (e: React.FormEvent<HTMLInputElement>) => void;
+};
```

Comment:
```
SelectOptionType & {
    onChange: (e: React.FormEvent<HTMLInputElement>) => void;
}
```

Reply:
ok

Reply:
i checked this more thoroughly and think that name prop needs to be there because inputs need it as props.

Reply:
my bad. mixed "label" and "name". Seems like LanguagedInputBind has also disabled: boolean; which could be added to the type.